credentials/src/main/kotlin/web5/sdk/credentials/model/VcDataModel.kt

```diff
@@ -0,0 +1,473 @@
+package web5.sdk.credentials.model
+
+import com.fasterxml.jackson.core.type.TypeReference
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import java.net.URI
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.TimeZone
+
+/**
+ * Global date format used for formatting the issuance and expiration dates of credentials.
+ * The value of the issuanceDate property MUST be a string value of an [XMLSCHEMA11-2] combined date-time
+ */
+public val DATE_FORMAT: SimpleDateFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'").apply {
+  timeZone = TimeZone.getTimeZone("UTC")
+}
+
+/**
+ * The [VcDataModel] instance representing the core data model of a verifiable credential.
+ *
+ * @see {@link https://www.w3.org/TR/vc-data-model/#credentials | VC Data Model}
+ */
+public data class VcDataModel(
```

Comment:
thoughts on using regular class instead of data class? 

reference: https://kotlinlang.org/docs/jvm-api-guidelines-backward-compatibility.html#don-t-use-data-classes-in-an-api

from this convo a _long_ time ago in our internal slack: https://sq-tbd.slack.com/archives/C05SXUR39FE/p1695165272574569

Reply:
I personally am in the data class camp. Less boilerplate, more kotlin magic.

If this is the only data class in our entire repo then yea I'm happy to conform and make it normal class.

Reply:
Updated 👍  no more data class